src/main/resources/assets/js/components/AppListComponent.jsx

```diff
@@ -12,17 +12,13 @@ var AppListComponent = React.createClass({
 
   propTypes: {
     collection: React.PropTypes.object.isRequired,
-    onSelectApp: React.PropTypes.func.isRequired
+    router: React.PropTypes.object.isRequired
```

Comment:
Yes, I also have a bigger refactoring task in mind. We should talk about that in the future all together.